src/ast/AstLlvmPrintVisitor.java

```diff
@@ -81,6 +83,7 @@ public void visit(MainClass mainClass) {
     @Override
     public void visit(MethodDecl methodDecl) {
         // TODO EX2 - print declaration
+        builder.append(methodDecl);
```

Comment:
I don't think this is the decleration.
It should be 'declare ...' in llvm format

Reply:
"define RETTYPE @CLASSNAME.METHODENAME(i8* %this, OPTIONAL-MORE-ARGS)"